docs/serializations.md

```diff
@@ -81,11 +81,58 @@ more serialized forms of itself.
 
 When serializing a physical SpdxDocument, any property of the logical element
 that can be natively represented within the chosen serialization format
-(e.g., `@context` prefixes in JSON-LD instead of the namespaceMap) may utilize
+(e.g., `@context` prefixes in JSON-LD instead of the namespaceMap) MAY utilize
```

Comment:
I don't know if we need the emphasis of capitalization on this or the next 2 changes - but I don't feel strongly if others prefer caps.

Reply:
I put it lowercase now.